packages/aws-cdk-lib/aws-eks/lib/alb-controller.ts

```diff
@@ -241,6 +241,27 @@ export interface AlbControllerOptions {
    * @default - Corresponds to the predefined version.
    */
   readonly policy?: any;
+
+  /**
+   * Override values to be used by the chart.
+   * For nested values use a nested dictionary. For example:
+   * values: {
+   *  autoscaling: false,
+   *  ingressClassParams: { create: true }
+   * }
+   *
+   * Note that the following values are set by the controller and cannot be overridden:
+   * - clusterName
+   * - serviceAccount.create
+   * - serviceAccount.name
+   * - region
+   * - vpcId
+   * - image.repository
+   * - image.tag
```

Comment:
```suggestion
   * The values to be used by the chart.
   * For nested values use a nested dictionary. For example:
   * values: {
   *  autoscaling: false,
   *  ingressClassParams: { create: true }
   * }
   *
   * Note that the following values are set by the controller and cannot be overridden:
   *  - clusterName
   *  - serviceAccount.create
   *  - serviceAccount.name
   *  - region
   *  - vpcId
   *  - image.repository
   *  - image.tag
```
Technically, we're not overriding the values.